src/lib/Repository/VersionService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository;
+
+use eZ\Publish\API\Repository\ContentService;
+
+/**
+ * Service for allowing deletion of versions.
+ */
+class VersionService
```

Comment:
I'm not sure, I personally don't like the idea of prefixing classes unless they are needed. The reason the others are prefixed is to not confuse them with the ones in the kernel. As a `Version` service does not exist it doesn't technically need it. Do you think it's better to prefix them all for consistency or just the ones that are needed?